Added a new topic (#2089)
github/explore: topics/perovskite/index.md

---

> aliases: perovskite

Reviewer: You shouldn't need to specify `aliases:` since this is the topic itself.

Author: ok, I get the point.

created_by: Paramesh Chandra

> display_name: Perovskite

Reviewer: ```suggestion
display_name: Perovskite
logo: perovskite.png
```

I think this will make the last linter check happy?

> short_description: An material science research branch which deals with a specialy structured material and exploring its properties. 

Reviewer: Better, thanks 👍🏻 

A little more spelling/grammar workshopping while we're at it:

```suggestion
short_description: A material science research branch which deals with exploring the properties of a specially structured material. 
```

> short_description: A perovskite is any material with a crystal structure similar to the mineral called perovskite, which consists of calcium titanium oxide (CaTiO3).

Reviewer: Instead of providing the definition here, could you clarify what repositories in this topic are actually doing and why they exist? Maybe something like this:

> Material science research related to the mineral perovskite, which may be used to...

Author: I have changed the short description accordingly.

topic: perovskite
wikipedia_url: https://en.wikipedia.org/wiki/Perovskite
---

> A perovskite is any material with a crystal structure similar to calcium titanium oxide (CaTiO3). This kind of material is widely used for photovoltaic applications and emerging as a new solar cell material. 

Reviewer: Oh, cool 😄
